src/infill.cpp

```diff
@@ -4,9 +4,20 @@
 #include "utils/polygonUtils.h"
 #include "utils/logoutput.h"
 
-namespace cura {
+/*!
+ * Function which returns the scanline_idx for a given x coordinate
+ *
+ * For negative \p x this is different from simple division.
+ *
+ * \warning \p line_width is assumed to be positive
+ *
+ * \param x the point to get the scansegment index for
+ * \param line_width the width of the scan segments
+ */
+static inline int computeScanSegmentIdx(int x, int line_width);
```

Comment:
This function is only used in this C++ file, so the scope of this function is here only. Putting it in the header function will make other modules who import this header file aware of the function, which is not necessary. This is why I moved it into this file.